src/lolex-src.js

```diff
@@ -493,8 +493,9 @@ function withGlobal(_global) {
     if (requestAnimationFramePresent) {
         timers.requestAnimationFrame = _global.requestAnimationFrame;
     }
+
     if (queueMicrotaskPresent) {
-        timers.queueMicrotask = _global.queueMicrotask;
+        timers.queueMicrotask = true;
```

Comment:
Why is this set to true?

Reply:
It gives another warning for just `var lolex = require("lolex");`.
Assigned native method not used later. So it's just for "yes, this feature supported". And actually we can remove it at all, because we filtering this method later inside install function to prevent faking by default. Correct me, please, if I'm wrong.

Reply:
I think this is here for restoring the original queueMicrotask later

Reply:
I thought so, but during `uninstall` function it restores original method from `_queueMicrotask` which was assigned inside `hijackMethod` from `target[method]` (`_global[queueMicrotask` in our case). `lolex.timers` not involved in this at all.

Example code on node 11.9.0:
```
const lolex = require("lolex");
var clock = lolex.install({ toFake: ["setTimeout", "queueMicrotask"] });

function microtask() {
  console.log("microtask");
}
function task() {
  console.log("task");
}

setTimeout(task, 0);
queueMicrotask(microtask);

clock.tick();

clock.uninstall();

setTimeout(task, 0);
queueMicrotask(microtask);
```
Outputs
```
microtask // faked method
task // faked method
(node:27136) ExperimentalWarning: queueMicrotask() is experimental.
    at get (internal/bootstrap/node.js:771:15)
    at hijackMethod (/Users/shdq/Projects/lolex-warning/node_modules/lolex/src/lolex-src.js:486:33)
    at Object.install (/Users/shdq/Projects/lolex-warning/node_modules/lolex/src/lolex-src.js:995:9)
    at Object.<anonymous> (/Users/shdq/Projects/lolex-warning/index.js:3:19)
    at Module._compile (internal/modules/cjs/loader.js:734:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:797:12)
microtask // native method
task // native method
```
It gives this warning on line 2, because there in hijackMethod `_queueMicrotask = _global[queueMicrotask]`;

I'm not insisting on `true`, we can even remove it at all, because this method is not going to be faked by default after this PR, but maybe it will be against whole concept of storing all the supported timers in `lolex.timers`. What do you think?